deploy/Vagrantfile

```diff
@@ -0,0 +1,91 @@
+# -*- mode: ruby -*-
+# vi: set ft=ruby :
+
+# All Vagrant configuration is done below. The "2" in Vagrant.configure
+# configures the configuration version (we support older styles for
+# backwards compatibility). Please don't change it unless you know what
+# you're doing.
+Vagrant.configure(2) do |config|
+  # The most common configuration options are documented and commented below.
+  # For a complete reference, please see the online documentation at
+  # https://docs.vagrantup.com.
+
+  ### COMMON VARIABLES
+  
+  VAGRANT_ROOT = File.dirname(File.expand_path(__FILE__))
+
+  ### SHARED COMPONENT
+  
+  # GlusterFS File Server 1
+  disk_file_1 = File.join(VAGRANT_ROOT, 'gfs_brick_1.vdi')
+  config.vm.define :file_server_1 do |fs_config|
+    fs_config.vm.box = "chef/centos-6.5"
```

Comment:
Is this notation compatible with older versions of vagrant? Would adding box_url setting make the Vagrantfile more broadly compatible?